### PR TITLE
feat: group all stories into a single viewable Storybook

### PR DESCRIPTION
**Pull request by m-davoodi**

### CODE REVIEW

File: libs/storybook/.storybook/main.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const rootMain = require('../../../.storybook/main');
+
+module.exports = {
+  ...rootMain,
+
+  core: { ...rootMain.core, builder: 'webpack5' },
+
+  stories: [...rootMain.stories, '../../**/*.stories.mdx', '../../**/*.stories.@(js|jsx|ts|tsx)'],
+  addons: [...rootMain.addons, '@nrwl/react/plugins/storybook'],
+  webpackFinal: async (config, { configType }) => {
```
Comment (mh-alahdadian) [inline libs/storybook/.storybook/main.js]: you are not modify webpack config so don't need webpackFinal

File: libs/storybook/.storybook/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "emitDecoratorMetadata": true,
+    "outDir": ""
+  },
+  "files": [
+    "../../../node_modules/@nrwl/react/typings/styled-jsx.d.ts",
+    "../../../node_modules/@nrwl/react/typings/cssmodule.d.ts",
+    "../../../node_modules/@nrwl/react/typings/image.d.ts"
+  ],
```
Comment (mh-alahdadian) [inline libs/storybook/.storybook/tsconfig.json:11]: what is this for ?!

File: libs/storybook/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    "jsx": "react-jsx",
+    "allowJs": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "forceConsistentCasingInFileNames": true,
+    "strict": true,
+    "noImplicitOverride": true,
+    "noPropertyAccessFromIndexSignature": true,
+    "noImplicitReturns": true,
+    "noFallthroughCasesInSwitch": true
```
Comment (mh-alahdadian) [inline libs/storybook/tsconfig.json:13]: most of these options should be in base config

File: libs/storybook/tsconfig.spec.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  "include": [
+    "jest.config.ts",
+    "**/*.test.ts",
+    "**/*.spec.ts",
+    "**/*.test.tsx",
+    "**/*.spec.tsx",
+    "**/*.test.js",
+    "**/*.spec.js",
+    "**/*.test.jsx",
+    "**/*.spec.jsx",
```
Comment (mh-alahdadian) [inline libs/storybook/tsconfig.spec.json:17]: use single glob pattern to match these `**/*.{test,spec}.{ts,tsx}`

**Review by mh-alahdadian — CHANGES_REQUESTED** [review]

File: libs/storybook/.storybook/tsconfig.json (hunk 2 above)
Comment (m-davoodi) [reply libs/storybook/.storybook/tsconfig.json:11]: these are generated files

File: .github/workflows/chromatic.yml [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # Required to retrieve git history
+      - name: Use Node.js version 16
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+          cache: 'yarn'
+      - run: yarn install --frozen-lockfile
```
Comment (mh-alahdadian) [inline .github/workflows/chromatic.yml]: yarn3 doesn't have --frozen-lockfile

**Review by mh-alahdadian — CHANGES_REQUESTED** [review]

**Review by mh-alahdadian — APPROVED** [review]

**Pull request merged**